### PR TITLE
Complete 3 tasks, task pack 3

### PR DESCRIPTION
**Pull request by ForceAwakenz**

### CODE REVIEW

File: task8.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    job: {
+      name: 'Andersen',
+      hiringDate: '23.07.2020'
+    }
+  }
+  
+// необходимо осуществить рефакторинг используя деструктуризацию над функцией без потери функциональности.
+
+function printJobName(user) {
+  const { job:
```
Comment (StanislawVictorovich) [inline task8.js:14]: Don't use additional variables. Do it with input parameters directly

File: task9.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// 3. Задача на Map, Set, WeakMap, WeakSet. Есть массив объектов. У каждого объекта есть id 
+const arr = [ { id: 1 }, { id: 2 }, { id: 3 }, { id: 1 }]
+// В массиве могут быть дубликаты. Написать функцию, которая будет удалять дубликаты из массива . Использовать вышеперечисленные инструменты для решения задачи. В дополнение к этому - разрешается использовать любые средства ES6.
+
+let filteringSet = new Set();
+
+for (let i = 0, counter = 0; i < arr.length; i++) {
+    counter = filteringSet.size;
```
Comment (StanislawVictorovich) [inline task9.js:8]: Do it inside a function. It should receive and return an array

File: task8.js (hunk 1 above)
Comment (ForceAwakenz) [reply task8.js:15]: Looks like it fixed, if I understood you correctly.

File: task9.js (hunk 2 above)
Comment (ForceAwakenz) [reply task9.js]: Done

File: task8.js (hunk 1 above)
Comment (StanislawVictorovich) [reply task8.js:15]: no, still see that you are using additional variables. In the function parameters you can see 'user'. Please make destructuring directly there

Comment (ForceAwakenz) [reply task8.js]: oh, now I got it!)

File: task8.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    job: {
+      name: 'Andersen',
+      hiringDate: '23.07.2020'
+    }
+  }
+  
+// необходимо осуществить рефакторинг используя деструктуризацию над функцией без потери функциональности.
+
+function printJobName( { job: { name } })
+{
```
Comment (StanislawVictorovich) [inline task8.js:14]: move this curly bracket to the previous line

Comment (ForceAwakenz) [reply task8.js]: Done.

**Review by StanislawVictorovich — APPROVED** [review]

**Pull request merged**